.travis.yml

```diff
@@ -25,6 +25,7 @@ env:
   global:
     - secure: "a7sD9iwPJJn3Fj+mn62GAmy/PEguh3elrilsp1KS+WfDiCiIKD8Q5KG2Jv67DGcQAGI3dPWeh7+ZhZ/W7nEipwWUBmSvGYVeoF63y8j6mNRLeekqspj94l47hXyFePj9bCadY1b1/xY4lE1pMEU8eA8AOUHUqCSuH+Kk/MuvyLM="
     - JAVA_OPTS: "-XX:+TieredCompilation -XX:TieredStopAtLevel=1 -Xms60M -Xmx720M -XX:InitialCodeCacheSize=40M -XX:ReservedCodeCacheSize=120M"
+    - JRUBY_OPTS: "-Xcext.enabled=true --debug"
```

Comment:
@olleolleolle travisci is complaining this is an unknown option. went ahead with the merge for the other work.

Reply:
I'll see what I can do about the unknown option, do you have a link to the failure handy?

Reply:
https://travis-ci.org/github/slim-template/slim/jobs/686709572#L635

Aha, jruby: the cext property not present - cool, dropping.